pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/plan/server/ServerPlanRequestUtils.java

```diff
@@ -280,7 +280,9 @@ private static void updateQueryOptions(PinotQuery pinotQuery, OpChainExecutionCo
       pinotQuery.setQueryOptions(queryOptions);
     }
     queryOptions.put(CommonConstants.Broker.Request.QueryOptionKey.TIMEOUT_MS,
-        Long.toString(executionContext.getDeadlineMs() - System.currentTimeMillis()));
+        Long.toString(executionContext.getActiveDeadlineMs() - System.currentTimeMillis()));
+    queryOptions.put(CommonConstants.Broker.Request.QueryOptionKey.EXTRA_PASSIVE_TIMEOUT_MS,
+        Long.toString(executionContext.getPassiveDeadlineMs() - executionContext.getActiveDeadlineMs()));
```

Comment:
The passive timeout isn't being used in the single-stage engine right?

Reply:
No, but it could in the future